ts/components/InstabugChatsComponent.tsx

```diff
@@ -59,48 +55,29 @@ type State = {
  * The icon has a badge if there are unread messages from the assistance
  */
 class InstabugChatsComponent extends React.PureComponent<Props, State> {
-  private handleIBChatPress = () => {
-    this.setState({
-      instabugReportType: some(BugReporting.reportType.question)
-    });
-    this.props.dispatchIBReportOpen(BugReporting.reportType.question);
-    openInstabugReplies();
-  };
-
   constructor(props: Props) {
     super(props);
     this.state = {
       instabugReportType: none,
       hasChats: false
     };
   }
-  public componentDidMount() {
-    // Register to the instabug dismiss event. (https://docs.instabug.com/docs/react-native-bug-reporting-event-handlers#section-after-dismissing-instabug)
-    // This event is fired when chat or bug screen is dismissed
-    BugReporting.onSDKDismissedHandler(
-      (dismiss: dismissType, _: BugReporting.reportType): void => {
-        // Due an Instabug library bug, we can't use the report parameter because it always has "bug" as value.
-        // We need to differentiate the type of report then use instabugReportType
-        if (this.state.instabugReportType.isSome()) {
-          this.props.dispatchIBReportClosed(
-            this.state.instabugReportType.value,
-            dismiss
-          );
-          // when user dismisses instabug report (chat or bug) we update the unread messages counter.
-          // This is because user could have read or reply to some messages
-          this.props.dispatchUpdateInstabugUnreadMessagesCounter();
-        }
-      }
-    );
-    this.checkInstabugChats();
-  }
 
   private checkInstabugChats = () => {
     Replies.hasChats(hasChats => {
       this.setState({ hasChats });
     });
   };
 
+  public componentDidMount() {
+    this.checkInstabugChats();
```

Comment:
```suggestion
    Replies.hasChats(hasChats => {
      this.setState({ hasChats });
    });
```
Since it used only into `componentDidMount` we can avoid to create another class method